src/touchstone/utils/lib.py

```diff
@@ -4,6 +4,14 @@
 _logger = logging.getLogger("touchstone")
 
 
+def print_metadata_dict(uuid, d):
+    for k, v in d.items():
```

Comment:
We need to print a CSV header.
In addition, we also need to print the "where" field to identify the server/pod where the metadata was taken from. Resulting in something like below:

```
uuid, where, field, value 
b34fe2f0-9d40-553f-becc-3132d495f51e, uperf-server-0-80b78c16, value.Model name, Intel(R) Xeon(R) Platinum 8175M CPU @ 2.50GHz, 
b34fe2f0-9d40-553f-becc-3132d495f51e, uperf-server-0-80b78c16, value.Architecture, x86_64,                                                      
b34fe2f0-9d40-553f-becc-3132d495f51e, uperf-server-0-80b78c16, value.CPU(s), 4,   
etc.
```